postprocessing_variant_calls/maf/helper.py

```diff
@@ -193,6 +194,7 @@ def __init__(self, file_path, separator, header=None):
         self.data_frame = self.__read_tsv()
         self.__gen_id()
         self.tsg_genes = tsg_genes
+        self.keep_exonic = keep_exonic
```

Comment:
This will probably go away when you implement using a JSON file as the "rule file".